docs/source/management/index.md

```diff
@@ -133,83 +133,79 @@ The user groups for the SRE on Microsoft Entra ID will also be deleted.
 $ dsh shm teardown
 ```
 
-::::{admonition} Tearing down an SHM
+::::{warning} Tearing down an SHM
 All SREs associated with the SHM must be torn down before the SHM can be torn down.
-Tearing down the SHM deletes any remotely stored SRE configurations.
+Tearing down the SHM permanently deletes **all** remotely stored configuration and state data.
 ::::
 
 ## Managing data ingress and egress
 
 ### Data Ingress
 
-It is the data provider's responsibility to upload the data required by the safe haven.
-
-```{important}
-Any data ingress must be signed off by the {ref}`role_data_provider_representative`, {ref}`role_investigator` and referee (if applicable).
-```
+It is the data provider representative's responsibility to upload the data required by the safe haven.
 
 The following steps show how to generate a temporary write-only upload token that can be securely sent to the data provider, enabling them to upload the data:
 
-- In the Azure portal select `Subscriptions` then navigate to the subscription containing the relevant SHM
-- Search for the resource group: `shm-<SHM ID>-sre-<SRE ID>-rg`, then click through to the storage account called: `sh<first three letter of SHM ID>sre<first three letters of SRE ID>sensitivedata`
-- Click `Networking` under `Settings` and ensure that the data provider's IP address is one of those allowed under the `Firewall` header, then hit the save icon in the top left
-- From the `Overview` tab, click the link to `Data storage` and then `Containers` (in the middle of the page)
-- Click `ingress`
-- Click `Shared access tokens` under `Settings` and do the following:
-    - Under `Signing method`, select `User delegation key`
-    - Under `Permissions`, check these boxes:
-        - `Write`
-        - `List`
-    - Set a 24 hour time window in the `Start and expiry date/time` (or an appropriate length of time)
-    - Leave everything else as default and click `Generate SAS token and URL`
-    - Copy the `Blob SAS URL`
+- In the Azure portal select **Subscriptions** then navigate to the subscription containing the relevant SHM
+- Search for the resource group: **`shm-<YOUR_SHM_NAME>-sre-<YOUR_SRE_NAME>-rg`**, then click through to the storage account called: **`sh<first three letters of SHM name>sre<first three letters of SRE name>sensitivedata`**
```

Comment:
The "first three letters" thing is a bit inelegant. I'm not sure if that name is also garunteed. It might be better to just say the storage account ending with "sensitivedata"?

Reply:
Agreed, and oh, of course - that's much better.